src/applications/vaos/utils/timezone.js

```diff
@@ -107,10 +123,15 @@ export function getTimezoneAbbrFromApi(appointment) {
  *
  * @export
  * @param {string} id - Facility id
+ * @param {boolean} isUseBrowserTimezone - Flag to determine if browser timezone
+ * should be used when facility id is not found.
  * @returns Timezone abbreviation with daylight savings stripped. Example: 'CT'
  */
-export function getTimezoneAbbrByFacilityId(id) {
-  const matchingZone = getTimezoneByFacilityId(id);
+export function getTimezoneAbbrByFacilityId(id, isUseBrowserTimezone = false) {
+  const matchingZone = getTimezoneByFacilityId(id, isUseBrowserTimezone);
+
+  if (isUseBrowserTimezone && matchingZone.startsWith('GMT'))
+    return matchingZone;
```

Comment:
The call to 'getTimezoneByFacilityId' will try to map 'GMT' timezones and return a 'GMT' timezone if the map was unsuccessful.